packages/ibm-products/src/components/Datagrid/useDisableSelectRows.ts

```diff
@@ -0,0 +1,119 @@
+/*
+ * Licensed Materials - Property of IBM
+ * 5724-Q36
+ * (c) Copyright IBM Corp. 2020
+ * US Government Users Restricted Rights - Use, duplication or disclosure
+ * restricted by GSA ADP Schedule Contract with IBM Corp.
+ */
+
+import { Hooks, Row, RowPropGetter, TableRowProps } from 'react-table';
+import { DataGridTableInstance, DatagridRow } from './types';
+
+interface PropGetterMeta {
```

Comment:
it would be better to move this interface definition to `packages/ibm-products/src/components/Datagrid/types/index.ts` so that it can be used by other Hooks as well, if required.